RouteSuggesterData/src/main/java/data/db/Fourteener.java

```diff
@@ -1,4 +1,9 @@
 package data.db;
 
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "fourteener_routes")
 public class Fourteener {
```

Comment:
Should this be `FourteenerRoute`? Ignore if this was changed in followup PR, I have a feeling it might have been

Reply:
It wasn't. Updated.